src/main/java/org/opentripplanner/middleware/utils/NotificationUtils.java

```diff
@@ -167,6 +169,7 @@ static String sendPush(
                 jsonBody
             );
             if (httpResponse.status == 200) {
+                LOG.info("Push notification sent");
```

Comment:
Would it be good to add some more dynamic details to this log message? Maybe notification ID + user?

Reply:
similar to the email part

Reply:
Good suggestion, added in https://github.com/ibi-group/otp-middleware/pull/318/commits/ddb31dd2c1d14e6542fc52c9f67a1ff0de557c83. I suppose we will have to find a way to clean up all these API calls.